<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crear_seccion</class>
 <widget class="QDialog" name="crear_seccion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>468</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>415</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>468</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f6fa;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widgetPrincipal" native="true">
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: #F7F9FC;
	color: #2d2d2d;
}
QLineEdit {
    border: 2px solid #2980b9;
    border-radius: 10px;
    padding: 2px 5px;
    background-color: white;
	color: #2d2d2d;
}
QPushButton {
	background-color: #2980b9;
    color: #FFFFFF;
    border: none;
    padding: 4px 6px;
    border-radius: 12px;
}
QPushButton:hover {
	background-color: #0D47A1
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLabel" name="lblTitulo_login">
        <property name="minimumSize">
         <size>
          <width>241</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>21</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Crear nueva sección</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QFrame" name="frameRol_login_4">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: white;
	border: 1.5px solid #1e88e5;
	border-radius: 10px;
}
QComboBox{
    border-radius: 10px;
    background-color: white;
    color: #333;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QComboBox" name="cbxNivel_crear_seccion">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>5</y>
           <width>201</width>
           <height>26</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>201</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>201</width>
           <height>26</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Nivel educación</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Inicial</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Primaria</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QFrame" name="frameRol_login_2">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: white;
	border: 1.5px solid #1e88e5;
	border-radius: 10px;
}
QComboBox{
    border-radius: 10px;
    background-color: white;
    color: #333;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QComboBox" name="cbxGrado_crear_seccion">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>5</y>
           <width>201</width>
           <height>26</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>201</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>201</width>
           <height>26</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Grado</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1er nivel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2do nivel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3er nivel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1er grado</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QFrame" name="frameRol_login_3">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: white;
	border: 1.5px solid #1e88e5;
	border-radius: 10px;
}
QComboBox{
    border-radius: 10px;
    background-color: white;
    color: #333;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QComboBox" name="cbxLetra_crear_seccion">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>5</y>
           <width>201</width>
           <height>26</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>201</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>201</width>
           <height>26</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Letra</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLineEdit" name="lneSalon_crear_seccion">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::ClickFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Salón/Aula</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLineEdit" name="lneCupo_crear_seccion">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::ClickFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Cupo</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QPushButton" name="btnAsignar_materias">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:{
background-color: #2980b9;
}
QPushButton:hover {
	background-color: #0D47A1
}</string>
        </property>
        <property name="text">
         <string>Asignar materias</string>
        </property>
        <property name="icon">
         <iconset resource="../icons/resources_ui.qrc">
          <normaloff>:/icons/nueva_materia.png</normaloff>:/icons/nueva_materia.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QPushButton" name="btnCrear_seccion">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #27ae60;
    color: #FFFFFF;
    border: none;
    padding: 6px 6px;
    border-radius: 14px;
}
QPushButton:hover {
	background-color: rgb(30, 124, 70);
}</string>
        </property>
        <property name="text">
         <string>Crear</string>
        </property>
        <property name="icon">
         <iconset resource="../icons/resources_ui.qrc">
          <normaloff>:/icons/confirm_white.png</normaloff>:/icons/confirm_white.png</iconset>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QPushButton" name="btnCancelar_crear_seccion">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
color: #2980b9;
border: 1.5px solid #2980b9;</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
        <property name="icon">
         <iconset resource="../icons/resources_ui.qrc">
          <normaloff>:/icons/cancelar_b2.png</normaloff>:/icons/cancelar_b2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/resources_ui.qrc"/>
 </resources>
 <connections/>
</ui>
